news/2017-07-03-scorewatch-june-week-4.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Scorewatch: June Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+date: 2017-07-03 02:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+---
+ 
+Many impressive first-ever-done scores, first places and more in this week’s article! 
+ 
+![](https://puu.sh/wA4Nj/d299c2b68d.png)
+
+[Yaong](https://osu.ppy.sh/u/1883865) entered the 600pp club in style with the first ever FC on [DragonForce - Soldiers of the Wasteland](https://osu.ppy.sh/b/962194), taking a 99.81% no mod FC to earn 639pp. Unbelievable consistency and accuracy from the Korean, this is a must watch!
+
+<iframe width="100%" height="315"
+src="https://www.youtube.com/embed/5AH-Vkcenc0" frameborder="0" allowfullscreen></iframe></br>
+
+[Mafham](https://osu.ppy.sh/u/3660531) may have just set one of the craziest EZDT plays of all time by grabbing an insane 97.11% EZDT FC on [Frederic - oddloop](https://osu.ppy.sh/b/1137879?m=0) on the oldloop difficulty. This is some top notch levels of reading and reflexes. Give this man his views!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/dOTlAs9lK4I" frameborder="0" allowfullscreen></iframe></br>
+
+[-GN](https://osu.ppy.sh/u/895581) might have churned out the craziest osu! EZHD play of all time, pulling off an absolutely terrifying 96.56% EZHD FC on [Nico Nico Douga - BARUSA of MIKOSU](https://osu.ppy.sh/b/24722) (which is 9.53*!), taking the No. 1 spot on the map live at Dreamhack. This would be worth approximately 707pp. What else can be better than for you to watch this amazing play yourself, together with the live chat, straight from -GN’s YouTube channel!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/MyTOUzA-sS8" frameborder="0" allowfullscreen></iframe></br>
+
+[Dusk](https://osu.ppy.sh/u/533210) is serious about cementing No. 1 Global as he continues his osu!catch score runs with an absurd HR SS on [LeaF - Evanescent](https://osu.ppy.sh/b/1009780) to gain a whopping 743pp and the No. 2 spot on the map, second only to [[224]Hyperw7](https://osu.ppy.sh/u/4158549) who was 1 100 short of SSing it himself. Sit back, relax and have a look at Rowlet sprinting across the screen with joy.
```

Comment:
``No . 1 Global`` -> his number one place in the global rankings
``No. 2`` -> number two
comma after hyperw7 hyperlink

maybe specify "rowlet catcher" or something so people don't get confused as to wtf a rowlet is